inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/tenant/InlongTenantService.java

```diff
@@ -59,4 +59,11 @@ public interface InlongTenantService {
      */
     Boolean update(InlongTenantRequest request);
 
+    /**
+     * List tenant by current user
+     * @param request
+     * @return
+     */
+    PageResult<InlongTenantInfo> listTenantByUser(InlongTenantPageRequest request);
```

Comment:
Method name does not match passed parameters。

Reply:
@vernedeng Suggest reuse the method of `listByCondition `, and put the user info into the `InlongTenantPageRequest` like `Response<PageResult<InlongGroupBriefInfo>> listBrief(@RequestBody InlongGroupPageRequest request)` does.

Reply:
It's a quite complex situation. 

For INLONG_ADMIN or INLONG_OPERATOR, they can aquire all tenant infomation, just set tenant list to null. 

But for other users, they can only aquire the tenant info which they belong to, hence we should list the tenants first by **_TenantRoleService::listTenantByUsername_**, and put the result in InlongTenantPageRequest.

Finally, they all invoke selectByCondition to aquire target tenant infos.